include/matx/transforms/fft.h

```diff
@@ -608,7 +609,7 @@ class matxFFTPlan2D_t : public matxFFTPlan_t<OutTensorType, InTensorType> {
       auto i_shape = i.Shape();
       // Get total number of batches
       size_t total_iter = std::accumulate(i_shape.begin(), i_shape.begin() + InTensorType::Rank() - batch_offset, 1, std::multiplies<shape_type>());
-      for (size_t iter = 0; iter < total_iter; iter++) {
+      for (size_t iter = 0; iter < total_iter-1; iter++) {
```

Comment:
I think rather than modifying this loop total_iter should be modified above maybe? It should be returning the product of all the dims you want to batch over, so I'm not sure I understand the `-1`.

Reply:
that -1 is a bug.  I was playing with that to see why we were hitting OOB issues.  I'll fix this.

Reply:
fixed